check_diff/src/lib.rs

```diff
@@ -335,3 +340,25 @@ pub fn compile_rustfmt(
         feature_runner,
     });
 }
+
+pub fn check_diff(config: Option<Vec<String>>, runners: CheckDiffRunners) -> i32 {
+    let mut errors = 0;
+    for entry in WalkDir::new(".").into_iter().filter_map(|e| e.ok()) {
+        let path = entry.path();
+        if path.is_file() && path.extension().map_or(false, |ext| ext == "rs") {
```

Comment:
I don't think we can call `WalkDir::new(".")`. We need to pass the path for the repository that we want to run the `check_diff` function on. Remember that we're going to clone various 3rd party repos to run the check-diff on them. We should update the function parameters of `check_diff` to take the path we want to search in.

Also, I still think it would be great if we abstracted the `.rs` file search into a function that returned an Iterator of `Path` or `PathBuf` objects so that `check_diff` isn't responsible for that logic. Plus it'll be a lot easier to unit test that we're searching for all `.rs` files within a directory if we break it out.